src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1450,8 +1456,28 @@ def _configure_security_policy_storage_params(arg_ctx):
                    ' new database. Must be greater than or equal to the source database\'s'
                    ' earliestRestoreDate value. ' + time_format_help)
 
-    with self.argument_context('sql midb list') as c:
-        c.argument('managed_instance_name', id_part=None)
+    with self.argument_context('sql midb retention-policy update') as c:
+        create_args_for_complex_type(
+            c, 'parameters', ManagedDatabase, [
+                'deleted_time',
+                'retention_days'
+            ])
+
+        c.argument('deleted_time',
+                   options_list=['--deleted-time'],
+                   help='If specified, updates retention days for a deleted database, instead of an existing database.'
+                   'Must match the deleted time of a deleted database on the source Managed Instance.')
+
+        c.argument('retention_days',
+                   options_list=['--retention-days'],
+                   required=True,
+                   help='New retention policy in days.')
```

Comment:
Can you give a more descriptive help here? and is there a valid range for the days?

Reply:
Sure I can add additional information here.